src/Illuminate/Routing/ResponseFactory.php

```diff
@@ -111,6 +111,33 @@ public function stream($callback, $status = 200, array $headers = [])
         return new StreamedResponse($callback, $status, $headers);
     }
 
+    /**
+     * Return a new streamed response as a file download from the application.
+     *
+     * @param  \Closure  $callback
+     * @param  string  $name
```

Comment:
`string|null` according to your method signature

Reply:
Hi @mfn 

Thanks for the PR review. I have pretty much copied the doc block from the "download" method. Would you also like me to make the same change to that doc block as well?